jenkins/deploy-clio-server.sh

```diff
@@ -75,7 +75,7 @@ CLIO_LOGBACK_CONF=clio-logback.xml
 # name of the env file used to set up clio with docker-compose
 CLIO_ENV_FILE=clio.env
 # port to expose for clio on the host VM
-HOST_CLIO_PORT=80
+HOST_CLIO_PORT=8000
 # port to expose for clio in the clio container
 CONTAINER_CLIO_PORT=${HOST_CLIO_PORT}
```

Comment:
@jacarey to answer your question about exposed port vs. port we run on, we currently don't run on 8080 when deployed. This variable [sets](https://github.com/broadinstitute/clio/blob/develop/clio-server/config/clio.conf.ctmpl#L4) the port Clio runs on in its container, and we handle binding it to the port exposed on the host through docker-compose. I set it to match the host port for convenience.